job/src/main/java/org/jenkinsci/plugins/workflow/job/AfterRestartTask.java

```diff
@@ -33,15 +33,18 @@
 import hudson.model.queue.CauseOfBlockage;
 import java.io.IOException;
 import org.acegisecurity.Authentication;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
 
 /**
  * Represents a {@link WorkflowRun} still running after a Jenkins restart.
  * Could be a {@code ContinuedTask}, though not really necessary since it is a {@link FlyweightTask} which would never be blocked anyway.
  */
+@ExportedBean
 class AfterRestartTask extends AbstractQueueTask implements Queue.FlyweightTask, Queue.TransientTask {
 
-    private final WorkflowRun run;
-    
+    @Exported private final WorkflowRun run;
```

Comment:
Maybe better to implement the public getter method. Is there any reason to hide this field?